Extensions/dist/extensions.js

```diff
@@ -157,3 +167,11 @@ function writeListFile(extensions) {
   });
   writePageFile('list.json', JSON.stringify(list));
 }
+
+if (require.main === module) {
```

Comment:
Not really a fan of this.  We should probably split out an executable that just calls `extBuilder.build(...)` with the correct args instead of doing a dance with global variables and this check.